software/salmon/quant/main.nf

```diff
@@ -41,11 +41,16 @@ process SALMON_QUANT {
         input_reads = "-a $reads"
     }
 
-    def strandedness =  ''
-    if (lib_type) {
+    def strandedness_opts = [
+        'A', 'U', 'SF', 'SR',
+        'IS', 'IU' , 'ISF', 'ISR',
+        'OS', 'OU' , 'OSF', 'OSR',
+        'MS', 'MU' , 'MSF', 'MSR' 
+    ]
+    def strandedness =  'A'
```

Comment:
I did not default to `A` since I thought it could be dangerous in the case the downstream code does not work correctly. The library type would then be guessed by 'salmon` and this would be unnoticed.

Reply:
I think Salmon auto-detecting by default would be fine rather than the pipeline breaking too?